paket.dependencies

```diff
@@ -0,0 +1,20 @@
+source https://www.nuget.org/api/v2/
+
+github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fsi
+github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fs
+github fsprojects/FSharp.TypeProviders.StarterPack src/DebugProvidedTypes.fs
+
+nuget FsCheck 0.9.2.0
```

Comment:
you should consider to remove the version restrictionss here. This is probably to restrictive.

usually we use the paket.lock file to pin the versions. If you keep the restrictions here, then `paket update`will always pull these versions.